backend_addon/{{ cookiecutter.__folder_name }}/bobtemplate.cfg

```diff
@@ -0,0 +1,5 @@
+[main]
+version = 6.0.13
+template = plone_addon
+git_init = True
+python = python3
```

Comment:
This is not necessarily a blocker, but would it be possible to add support for mr.bob/bobtemplates.plone to get their settings from pyproject.toml, so that we don't need more config files?

Reply:
I guess that there's no problem with that. Would you mind adding an issue in bobtemplates.plone repo?

Reply:
https://github.com/plone/bobtemplates.plone/issues/569

Reply:
release is out

Reply:
I'll resolve this since it isn't a blocker.